internal/command/command.go

```diff
@@ -148,6 +151,77 @@ func finalize(ctx context.Context) {
 	}
 }
 
+// applyAliases consolidates flags with aliases into a single source-of-truth flag.
+// After calling this, the main flags will have their values set as follows:
+//   - If the main flag was already set, it will keep its value.
+//   - If it was not set, but an alias was, it will take the value of the first specified alias flag.
+//     This will set flag.Changed to true, as if it were specified manually.
+//   - If none of the flags were set, the main flag will remain its default value.
+func applyAliases(ctx context.Context) (context.Context, error) {
+
+	var (
+		invalidFlagNames []string
+		invalidTypes     []string
+
+		flags = flag.FromContext(ctx)
+	)
+	flags.VisitAll(func(f *pflag.Flag) {
+		if f.Annotations == nil {
```

Comment:
no need to check a map is nil  https://go.dev/play/p/1tMleRQ225o

Reply:
could've sworn I've been bitten by that before